static/js/components/Footer.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './Footer.css';
+
+
+const Footer = props => (
+  <div className={styles.footer}>
+    <div className={styles.footerContent}>
+      SkyPortal version {props.sysinfo.skyportal_version}. Please file issues at&nbsp;
```

Comment:
Components should not know anything about the app state.  So this should simply be something like

```
const Footer = ({version}) => (
...
```